app/Http/Controllers/DesktopAuthController.php

```
@@ -0,0 +1,106 @@
+
+        return redirect()->intended(RouteServiceProvider::HOME);
+    }
+
+    #[Post('/login/{user}', name: 'desktop.login.store', middleware: ['guest', 'throttle:desktop-auth'])]
+    public function store(Request $request, User $user, VerifyDesktopUnlock $verifyDesktopUnlock): RedirectResponse
+    {
+        if ($this->lockedUserId($request) !== null) {
+            $verifyDesktopUnlock->verify($user, $request->all());
+        }
+
+        Auth::login($user);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Auth' in method 'store'. (undefined)

(StaticAccess)